src/models/hooks/activityReport.js

```diff
@@ -478,29 +481,6 @@ const automaticGoalObjectiveStatusCachingOnApproval = async (sequelize, instance
           individualHooks: true,
         },
       )));
-
```

Comment:
As far as I can tell, this is unnecessary - we find or create activity report objectives with the proper status every time we save, and we wouldn't want to update an ARO to in progress automatically given that can override the moment in time caching we are trying to achieve.